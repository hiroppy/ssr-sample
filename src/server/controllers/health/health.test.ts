import httpMocks, { MockResponse } from 'node-mocks-http';
import { get } from '.';

test('should return the get of api response', () => {
  const req = httpMocks.createRequest();
  const res = httpMocks.createResponse();
  const data = get(req, res) as MockResponse<ReturnType<typeof get>>;

  expect(data._getJSONData()).toMatchSnapshot();
});
