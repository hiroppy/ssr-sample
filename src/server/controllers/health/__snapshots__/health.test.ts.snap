// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the get of api response 1`] = `
Object {
  "heath": "ok",
}
`;
